README.md

```diff
@@ -2,42 +2,42 @@
 
 Informàtics Empordà és un grup creat per... si, per informàtics de l'Empordà!
 
-La finalitat del grup és ser punt de trobada de professionals i aficionats a les diferents branques de la informàtica.
+La finalitat del grup és ser punt de trobada de professionals i afeccionats a les diferents branques de la informàtica.
```

Comment:
Em sona una mica forçat, i segons diccionari.cat aficionat és correcde...

Reply:
@ctrl-alt-d @mgarcia96 com ho veieu? aficionats o afeccionats?

Reply:
@utrescu @ctrl-alt-d @mgarcia96 permeteu-me un ping, per desencallar el tema i així tancar la PR

Reply:
Que tal geek? Aquestes paraules modernes solen agradar

Reply:
ok @mgarcia96 doncs m'ho agafo com un lgtm 👍